### PR TITLE
[Notifier] Added MessageBirdOptions to the message-bird-notifier bridge

### PR DESCRIPTION
**Pull request by EJTJ3**
| Q             | A
| ------------- | ---
| Branch?       | 5.4 
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

Added a MessageBirdOptions to the message-bird-notifier. This allows adding validity and reference to the messages send with the notifier component.

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/MessageBird/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+* Added MessageBirdOptions
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/MessageBird/CHANGELOG.md]:
```suggestion
* Add `MessageBirdOptions`
```

**Comment by OskarStark** [comment]
Hi, thank you for working on this feature.
I am afraid it will not be merged, based on the same reasons in https://github.com/symfony/symfony/pull/39774 by @fabpot

File: src/Symfony/Component/Notifier/Bridge/MessageBird/MessageBirdOptions.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    {
+        return $this->options;
+    }
+
+    public function getRecipientId(): ?string
+    {
+        return $this->options['recipients'] ?? null;
+    }
+
+    public function validity(int $validity): self
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Notifier/Bridge/MessageBird/MessageBirdOptions.php:43]: please add `@return $this` annotations every where possible

File: src/Symfony/Component/Notifier/Bridge/MessageBird/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+5.4
+---
+
+* Add `MessageBirdOptions`
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Notifier/Bridge/MessageBird/CHANGELOG.md]: missing space at start of line

**Comment by EJTJ3** [comment]
@nicolas-grekas I've updated the codestyle.

**Comment by fabpot** [comment]
Closing as explained in https://github.com/symfony/symfony/pull/39774

**Pull request closed without merging**